packages/auth/migrations/20220502133210_create_test_table.js

```diff
@@ -0,0 +1,9 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('dummyTables', function (table) {
```

Comment:
I have it so that the build step doesn't break because it doesn't find any migration files. But we could merge in https://github.com/interledger/rafiki/pull/332 first and then rebase this branch off main to avoid having to do this.